packages/iov-bns/src/types.ts

```diff
@@ -93,10 +93,25 @@ export enum ProposalStatus {
   Withdrawn,
 }
 
+export enum VoteOption {
+  Yes,
+  No,
+  Abstain,
+}
+
+export interface TallyResult {
+  readonly totalYes: number;
+  readonly totalNo: number;
+  readonly totalAbstain: number;
+  readonly totalElectorateWeight: number;
```

Comment:
Is this just a sum of the other numbers? Do we need it at all?

Reply:
No, it is the total weight of eligable voters, so

```
voterParticipation = (totalYes+totalNo+totalAbstain) / totalElectorateWeight
```

The number is indeed redundant (backend stores it twice or even 3 times on purpose), but with data that is indirect (pseudo-code):

```
getElectorate(Proposal.electorateVersionedID).totalWeight
```